src/cmake/externalpackages.cmake

```diff
@@ -68,6 +68,7 @@ find_package (LLVM 3.9 REQUIRED)
 
 # ensure include directory is added (in case of non-standard locations
 include_directories (BEFORE SYSTEM "${LLVM_INCLUDES}")
+include_directories (BEFORE SYSTEM "${CLANG_INCLUDES}")
```

Comment:
This line failed in the CI tests.
CMake doesn't like it if the string is empty?